src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -4,13 +4,21 @@
 
 using Oceananigans.Grids: topology, min_Δx, min_Δy, min_Δz
 
-function min_Δxyz(grid)
+function min_Δxyz(grid, ::Type{ThreeDimensional})
```

Comment:
```suggestion
function min_Δxyz(grid, ::ThreeDimensional)
```

This will be possible if we have `direction(closure)` function. (Or maybe `directionality(closure)` or `isotropy(closure)`...)